docs/tutorials/01_KalmanFilterTutorial.py

```diff
@@ -102,6 +103,13 @@
 # And the clock starts
 start_time = datetime.now()
 
+# %%
+# We note that it can sometimes be useful to fix our random number generator in order to probe a
+# particular example repeatedly. That option is available by uncommenting the next lines.
+
+# import random
+# random.seed(1991)
```

Comment:
Just tested this and it's doesn't work. Need to use NumPy random seed. I'll change it and merge this PR.